test/Ionide.ProjInfo.Tests/Tests.fs

```diff
@@ -1241,13 +1243,21 @@ let debugTests toolsPath workspaceLoader (workspaceFactory: ToolsPath -> IWorksp
     ptestCase
     |> withLog (sprintf "debug - %s" workspaceLoader) (fun logger fs ->
 
-        let projPath = @"D:\Programowanie\Projekty\Ionide\dotnet-proj-info\src\Ionide.ProjInfo.Sln\Ionide.ProjInfo.Sln.csproj"
-
         let loader = workspaceFactory toolsPath
 
-        let parsed = loader.LoadProjects [ projPath ] |> Seq.toList
+        // let projPath = @"D:\Programowanie\Projekty\Ionide\dotnet-proj-info\src\Ionide.ProjInfo.Sln\Ionide.ProjInfo.Sln.csproj"
+        // let parsedProjs = loader.LoadProjects [ projPath ] |> Seq.toList
+
+        // printfn "%A" parsedProjs
 
-        printfn "%A" parsed
+        let slnPath = @"C:\Users\JimmyByrd\Documents\Repositories\public\TheAngryByrd\FsToolkit.ErrorHandling\FsToolkit.ErrorHandling.sln"
```

Comment:
Could revert

Reply:
eh, this test is a scratch-pad for folks anyway so I don't think the content is really special